### ISSUE TITLE
cloud agent cannot be initialized with mF2C_User nor mF2C_Pass

### ISSUE THREAD

**Issue by cjdcordeiro** [mF2C/ResourceManagement#54]
When deploying the cloud agent, there are no existing users in the DB as this is to be considered the first mf2c agent of them all.

Thus the identification module will fail to initialize

**Comment by alxjand** [comment]
Deploying the cloud agent for the very first time should be considered as an especial use case and, therefore, treated as such. I mean, we won't launch the cloud agent without any data every day, nor every month, and ideally, nor even every year.

Considering the above, I think we have two options so solve this issue:

1. Setting a fixed IDKey and deviceID for the cloud agent. Given the fact that both IDs will be know in advance, this option can be considered as insecure (even when we've talked before about this and the conclusion is that these strings are not sensitive). 

2. Using the RUN command to create the "admin" or "mf2c" user account. The system administrator should provide a valid email address, a username and password, and then to wait for CIMI and dataClay to be ready and execute the curl instruction with the proper parameters. I'd go for this option.

**Comment by cjdcordeiro** [comment]
I agree this is a special case, but I wouldn't say it is something that "won't happen" frequently. The cloud agent is always the first in the chain, and we can have multiple mf2c providers, with multiple clouds, and multiple cloud agents within the same cloud for redundancy.

For me, I'd go with the fastest one, which ever one that is...(even though yes, 2. would be better)

**Comment by sstanovnik** [comment]
Resolved through mf2c/resource-bootstrap@c7825fa3c82b4286d893b3fc42f6b86fe6162c9a and image version 1.3.0.

**Comment by ALEJANDROJ19** [comment]
Test failed. Identification result:

```
[POLICIES - debug_init] DEBUG     : Identification request result: {'status': '412', 'message': 'Invalid user credentials'}
```
